### ISSUE TITLE
Why is my validation loss lower than my training loss?

### ISSUE THREAD

**Issue by limaodaxia** [abhiwalia15/Why-is-my-validation-loss-lower-than-my-training-loss-#1]
hello, do you know the reason?
